src/api/status/src/views/builds.hbs

```diff
@@ -32,6 +32,43 @@
     </div>
   </nav>
   <div class="container-fluid py-2">
+    <div id="build-header" class="border rounded p-2 mb-3 bg-white">
```

Comment:
Can you make this one a [card](https://www.creative-tim.com/learning-lab/bootstrap/cards/material-dashboard)?